windows/devices/uwb/include/windows/devices/uwb/UwbConnector.hxx

```diff
@@ -68,24 +78,26 @@ public:
     // IUwbDeviceDdiConnector
     /**
      * @brief Sets the callbacks for the UwbDevice that owns this UwbConnector
+     * You can pass in a partially filled in struct, in which case this function returns a partially filled in struct of tokens corresponding to those callbacks.
      *
      * @param callbacks
-     * @return std::weak_ptr<::uwb::RegisteredCallbackToken> You can pass this pointer into DeregisterEventCallback to deregister this event callback
+     * @return ::uwb::UwbRegisteredDeviceEventCallbackTokens You can pass the pointers into DeregisterEventCallback to deregister the event callbacks
      */
-    virtual std::weak_ptr<::uwb::RegisteredCallbackToken>
-    RegisterDeviceEventCallbacks(std::weak_ptr<::uwb::UwbRegisteredDeviceEventCallbacks> callbacks) override;
+    virtual ::uwb::UwbRegisteredDeviceEventCallbackTokens

```

Comment:
Given we've now made all the callbacks individual, I wonder if we should expose individual functions to register them? We could keep this one for callers that want to aggregate, but it would be nice for callers that are only interested in 1 or 2 callbacks to just call individual functions.

Reply:
opened an issue to track this #231